src/components/home/index.jsx

```diff
@@ -91,15 +92,18 @@ const HomePage = () => {
           </div>
         </div>
         <footer className='footer'>
-          <a rel='noreferrer noopener' target='_blank' href='https://t.me/fuse_fi'>
+          <a rel='noreferrer noopener' target='_blank' href='https://t.me/voltage_finance'>
             <img src={telegram} />
           </a>
           <a rel='noreferrer noopener' target='_blank' href='https://github.com/fuseio/fusefi-interface'>
```

Comment:
can we also update the github

Reply:
Updated github to: https://github.com/voltfinance/